### PR TITLE
[client] Fix an untranslated label

### PR DESCRIPTION
**Pull request by ashie**
"Retry interval (sec)" in "Monitoring Servers" page isn't translated.

### CODE REVIEW

File: client/conf/locale/ja/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -615,7 +624,11 @@ msgstr "監視サーバートリガー再読込"
 msgid "Connection status"
 msgstr "通信状態"
 
-#: viewer/servers_ajax.html:81
+#: viewer/servers_ajax.html:69
+msgid "URL"
+msgstr ""
```
Comment (kz0817) [inline client/conf/locale/ja/LC_MESSAGES/django.po]: これが空白って意図的ですか？それとも漏れ？

File: client/conf/locale/ja/LC_MESSAGES/djangojs.po [hunk 2]
```diff
@@ -1692,10 +1739,21 @@ msgstr "アイテムが選択されていません！"
 msgid "Successfully saved."
 msgstr "正常に保存されました。"
 
-#: static/js/latest_view.js:151
+#: static/js/latest_view.js:154
 msgid "Graph"
 msgstr "グラフ"
 
+#: static/js/push_notify.js:22
```
Comment (kz0817) [inline client/conf/locale/ja/LC_MESSAGES/djangojs.po]: ここも未翻訳のようですが、このPRではスキップですか？

File: client/conf/locale/ja/LC_MESSAGES/djangojs.po [hunk 3]
```diff
@@ -98,14 +102,20 @@ msgstr "常駐"
...
-#: static/js/log_search_systems_view.js:112 static/js/servers_view.js:228
+#: static/js/log_search_systems_view.js:112 static/js/servers_view.js:225
 msgid "EDIT"
 msgstr "編集"
 
+#: static/js/before_close.js:8
+msgid ""
+"Hatohol is running to monitor system(s).\n"
+"You are trying to leave or close this page."
+msgstr ""
```
Comment (kz0817) [inline client/conf/locale/ja/LC_MESSAGES/djangojs.po]: ここも意図的にスキップですか？

**Comment by ashie** [comment]
このPRはあくまでも"Retry interval (sec)"をhatohol-server-type-utilに追加するだけの修正
（hatohol-server-type-utilの修正が必要で、こっちが本題）で、それ以外は手をつけていません。

**Comment by kz0817** [comment]
了解。
